### DISCUSSION TITLE
Changing the time to reset total_daily_energy values

### DISCUSSION THREAD

**Discussion by PhotonHebs** [emporia-vue-local/esphome#153] [Q&A] [answered]
Does anybody know if it Is possible to change the time that the total_daily_energy values are reset.   Midnight isn't ideal because my energy tariff changes at 0530hrs.  I'v though about changing the timezone, but there must be an easier way?

Is there a way of having two reset trigger times?  Say at 2330hrs & 0530hrs for example.  this is when my tariff changes from day to night rate.

**Comment by flaviut** [comment, marked as answer]
You shouldn't rely on these values resetting at any particular time; they also reset when you have a power outage.

Take a look at https://www.home-assistant.io/docs/energy/faq/#split-consumption-by-tariffs
